samples/python/tutorial_code/introduction/display_image/display_image.py

```diff
@@ -0,0 +1,18 @@
+## [includes]
+import cv2 as cv
+## [includes]
```

Comment:
includes => imports?

Reply:
I did this to be consistent with the C++ code

Reply:
I understant, but `@snippet` doesn't require that:

```.diff
-@snippet samples/python/tutorial_code/introduction/display_image/display_image.py includes
+@snippet samples/python/tutorial_code/introduction/display_image/display_image.py imports
```